configtx/application.go

```diff
@@ -458,7 +481,11 @@ func newApplicationGroupTemplate(application Application) (*cb.ConfigGroup, erro
 	applicationGroup := newConfigGroup()
 	applicationGroup.ModPolicy = AdminsPolicyKey
 
-	if err = setPolicies(applicationGroup, application.Policies, AdminsPolicyKey); err != nil {
+	if application.ModPolicy != "" {
+		applicationGroup.ModPolicy = application.ModPolicy
+	}
```

Comment:
This block is untested. Can you please add a test to assert that the presence of an `application.ModPolicy` overwrites the default value?